### DISCUSSION TITLE
Downloader Coapp not recognized after install M1 MacOS 14.4

### DISCUSSION THREAD

**Discussion by goldmahn** [aclap-dev/video-downloadhelper#289] [Help and Q&A]
### Your question

Hi there!
So suddenly I can't download anything.  I'm constantly being told the Downloader App isn't installed, even thought I've done so.  I've uninstalled, reinstalled, updated, everything I can think of, and still nothing.  Anyone have any ideas how I can fix this?  Addon and CoApp details attached.

Thanks in advance for any insights here!

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/115.0
coapp: no.

{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "arm64",
    "node": "18"
  },
  "home": "/Users/goldm1ni",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}

**Comment by Mattmjs** [comment]
Did you refer to the troubleshooting ? https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized#mac

**Reply by goldmahn** [reply]
Yes I did.  I tried starting, restarting, uninstalling, reinstalling, and restarting again, and while it the coapp said it was ready to be used, the downloader in my browser continues to report it isn't connected. 
<img width="379" alt="Screenshot 2024-03-15 at 10 02 08 PM" src="https://github.com/aclap-dev/video-downloadhelper/assets/14301865/d200f3e7-063b-4a3a-86e6-98930cbde7f9">

**Reply by Mattmjs** [reply]
So after install it says to **start the coapp only one time after installation** to re register it with the relevant browsers.
And you've done this ?

**Comment by goldmahn** [comment]
Yes, and while it says it is ready, my browser doesn’t recognize it.

**Reply by Mattmjs** [reply]
I'm not a Mac user and that is all it says to do, so I don't know what else it  might be.
Your M1 is Apple silicon based hardware not Intel and you've installed the relevant app for this hardware.
Am I right on all this ?

Do you also have an anti virus & firewall that might block the app.

**Comment by goldmahn** [comment]
Yes, it’s an M2, and the relevant app has been installed.  I’m currently running Malwarebytes as an antivirus, but it doesn’t blockade anything (honestly MacOS is more of a pain there), and my firewall is external hardware.  

I’m confident that nothing is blocking the app.  

Ari Goldman, laird
***@***.***

**Reply by Mattmjs** [reply]
I'd check anyway to make sure and then it can ruled out if it's ok, other than that I don't have any other ideas.
I'm just a user of the software like you.

**Reply by GrampaWildWilly** [reply]
Antivirus & firewall would typically be issues during download.  This user is just trying to install the product so he hasn't gotten that far yet.

Perhaps #291 might be helpful.  I'm also just another user & I also know squat about Mac.
